doc/source/conf.py

```diff
@@ -376,6 +376,14 @@
    os.chdir(r'{os.path.dirname(os.path.dirname(__file__))}')
 """
 
+warnings.filterwarnings(
```

Comment:
is this unrelated?

Reply:
Yes - I need to add this for the docs to build locally. Not sure why - something to do with my matplotlib installation. Will be removed when this is ready.